.github/workflows/test.yml

```diff
@@ -9,16 +9,17 @@ on:
 jobs:
   build:
     name: Continuous Testing
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Same comment as in the other PR, can we build/test in Mac and Windows?

Reply:
@rgnote - How about this be a separate 'feature' request?  It's not common IMO to build a golang library on multiple platforms - theoretically it should just work because it's golang.

Reply:
We can track this as separate request. We do need to test on multiple platforms because there are differences in implementation (E.g. [directory structure spec](https://github.com/notaryproject/notation/blob/main/specs/directory.md)) that need to be tested in each platform.

Reply:
Make sense. But we may want to do this in a seperate PR, as currently we use `make test` to generate coverage report, which will not work on Windows.